template/src/dummy.scss

```diff
@@ -1,3 +1,5 @@
+@use 'src/assets/stylesheets/application' as app;
```

Comment:
I am not sure about namespacing it to `app` as it makes it more verbose 🤔 

If we were to import Bootstrap, then that would be prefixed to `bs` (or an alternative). There is no concern for name clashing IMO.

Reply:
I see, removed in [7a8e13b](https://github.com/nimblehq/react-templates/pull/78/commits/7a8e13bf22b0afd7676a503a5789c7c90dcb89d2) at first I was just worried that the name is quite long 😅